README.md

```diff
@@ -29,4 +29,11 @@ Relevant Google/Apple or DP-3T protocol apps.
 1. SwissCovid (DP-3T): https://github.com/DP-3T/dp3t-app-android-ch
 2. Covid-Warn-App (GA): https://github.com/corona-warn-app
 3. Immuni (GA): https://github.com/immuni-app
+4. eRouška (bespoke): https://github.com/covid19cz/erouska-android
 
+## Build notes
+
+The protobuf code can be generated using the following in the project root.
+```
+sb2 protoc --cpp_out=contracd/src  contracd/src/contrac.proto
```

Comment:
As the protobuff files are generated on build time I don't think its really good that this is mentioned in the README.
Is this for debugging?